app/models/user.rb

```diff
@@ -5,7 +5,7 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable
   include DeviseTokenAuth::Concerns::User
   validates :email, presence: true
-  validates :password, presence: true, uniqueness: { case_sensitive: false }
+  validates :encrypted_password, presence: true, uniqueness: { case_sensitive: false }
```

Comment:
I haven't payed enough attention to this. `encrypted_password` should not be unique, two users can have the same hash

Reply:
Also devise validates its presence. Remove this line

Reply:
Removed it.